paddle/fluid/operators/math/depthwise_conv.cu

```diff
@@ -137,7 +117,12 @@ __device__ __inline__ void KernelDepthwiseConvCFilter(
           if (h_in >= 0 && h_in < input_height && w_in >= 0 &&
               w_in < input_width) {
             const int offset = in_offset + h_in * input_width + w_in;
-            value += r_weight[h_f * c_filter + w_f] * input_data[offset];
+            if (fuse_relu_before_conv) {
```

Comment:
if/els inside kernel seems will hurt performance a lot, did you make a comparison?

Reply:
`fuse_relu_before_conv` is a template variable, this will be optimized at compile time